command/job_status.go

```diff
@@ -406,29 +406,33 @@ func formatAllocListStubs(stubs []*api.AllocationListStub, verbose bool, uuidLen
 
 	allocs := make([]string, len(stubs)+1)
 	if verbose {
-		allocs[0] = "ID|Eval ID|Node ID|Task Group|Version|Desired|Status|Created At"
+		allocs[0] = "ID|Eval ID|Node ID|Task Group|Version|Desired|Status|Created At|Modified At"
```

Comment:
We should standardize on `Created` vs `Created At` and same with modify.

Reply:
I left it this way because it is grammatically correct to say "Created at yyyy:mm:dd" and "Created 2 minutes ago". The other phrasing `Created At x seconds ago` sounds wrong. I'd be fine with changing both to `Created`

Reply:
Yeah I vote drop the `At`